client/browser/src/browser-extension/after-install-page/AfterInstallPageContent.tsx

```diff
@@ -104,25 +104,56 @@ export const AfterInstallPageContent: React.FunctionComponent<React.PropsWithChi
                             <h2 className="mb-4">Make it work on your codehost</h2>
                             <div className="bg-2 rounded p-3 mb-3 d-flex flex-column justify-content-center">
                                 <h3 className={classNames('mb-3', styles.codeHostTitles)}>
-                                    <Icon className={styles.codeHostLogo} as={GithubIcon} /> github.com
+                                    <Icon
+                                        role="img"
+                                        className={styles.codeHostLogo}
+                                        as={GithubIcon}
+                                        title="Github logo"
```

Comment:
```suggestion
                                        aria-hidden={true}
```

This one can be hidden